dotnet-base-libraries/ReturnInsertedIdentityWithDapper/ReturnInsertedIdentityWithDapper/SetUp.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Dapper;
+using Microsoft.Data.SqlClient;
+
+namespace ReturnInsertedIdentityWithDapper;
+
+public static class SetUp
+{
+    public static void CreateTable(string connectionString)
+    {
+        using var connection = new SqlConnection(connectionString);
+
+        connection.Open();
+
+        var createTableSql = "CREATE TABLE Students (Id INT IDENTITY(1,1) PRIMARY KEY, " +
+            "Surname VARCHAR(255), " +
+            "Firstname VARCHAR(255))";
```

Comment:
Instead of concatenating multiline strings with `+` signs, can we use a more sophisticated approach like [raw string literals](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/raw-string)?

Reply:
Used string literal